recipes/fft/all/test_package/fft4f2dt.c

```diff
@@ -0,0 +1,107 @@
+/* test of fft4f2d.c */
+
+#include <math.h>
+#include <stdio.h>
+#include <fft/alloc.h>
+#include <fft/fft.h>
+#include <fft/fft2.h>
+#define MAX(x,y) ((x) > (y) ? (x) : (y))
+
+/* random number generator, 0 <= RND < 1 */
+#define RND(p) ((*(p) = (*(p) * 7141 + 54773) % 259200) * (1.0 / 259200))
+
+
+int main()
+{
+    void putdata2d(int n1, int n2, double **a);
```

Comment:
Does it really need 3 very big test packages?

Reply:
they are not so big, but at least test all 3 APIs are there

Reply:
I've decided to remove these existing samples and leave only one very simple